DropboxSync.BLL/DropboxSyncContext.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DropboxSync.BLL.Configurations;
+using DropboxSync.BLL.Entities;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DropboxSync.BLL
+{
+    public class DropboxSyncContext : DbContext
+    {
+        public DbSet<DossierEntity> Dossiers { get; set; }
+        public DbSet<ExpenseEntity> Expenses { get; set; }
+        public DbSet<InvoiceEntity> Invoices { get; set; }
+        public DbSet<UploadEntity> Uploads { get; set; }
+
+#pragma warning disable CS8618 // Un champ non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable.
```

Comment:
is there a possibility to fix the warning?

Reply:
Apparently yes. [This ](https://docs.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types#dbcontext-and-dbset) method will be used